spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,42 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def create_control_layout(controls, main_widget):

```

Comment:
@timhoffm I understand the concept that you try to illustrate, but in this case it is not making things better, it is mixing a lot of concepts, widgets, layouts, strings that have a specific meaning only if you used the right string, and the idea at the end is to make the code understandable to everyone and not only you in this case.

`create_control_layout(tools_layout, main_widget)` sounds better, but to be honest the general solution to this issue should be to handle the top controls (if any) at the plugin level. @rlaverde do you have some suggestions here?